svc-update/go.mod

```diff
@@ -6,5 +6,6 @@ require (
 	github.com/ODIM-Project/ODIM/lib-dmtf v0.0.0-20200821035243-eace91cf2147
 	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20200821035243-eace91cf2147
 	github.com/ODIM-Project/ODIM/svc-plugin-rest-client v0.0.0-20200824082927-a64d9b6849af
+	github.com/ODIM-Project/ODIM/svc-systems v0.0.0-20200825050041-8ecdee28f564
```

Comment:
remove the svc-system reference from go.mod

Reply:
Addressed